src/I18n/languages/fr.json

```diff
@@ -70,6 +71,8 @@
     "reportError": "Signaler Erreur"
   },
   "authentication": {
+    "noTenantFoundTitle" : "Aucune organisation trouvée",
+    "noTenantFoundMessage" : "Voulez-vous en ajouter une avec un QR-Code?",
```

Comment:
Not translated from english

Reply:
You mean the line 75 ?

Reply:
More like this:
Voulez-vous enregistrer une organisation en utilisant le QR-Code?

Reply:
Fixed